src/Cat.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Arrays;
+import java.util.Random;
+
+public class Cat extends Animal {
+    private static int catCounter;
+
+    public Cat(String name) {
+        super(name);
+    }
+
+    public static int getCatCounter() {
+        return catCounter;
+    }
+
+    public static String catRandomName() {
+        var random = new Random();
+        var list = Arrays.asList("Марсік", "Каспер", "Луна", "Барсік", "Юкі", "Лейла", "Ласка", "Ніка", "Боня", "Соня");
+        return list.get(random.nextInt(list.size()));
+    }
+
+    public static String oneCatResult() {
+        Cat cat = new Cat(catRandomName());
+        catCounter++;
```

Comment:
The correct place to increment your counter is a constructor, not a method! By incrementing your counter here you force to use this method in order to get the current result. I'm sorry, but this concept is incorrect. Please move this call to the constructor of this class.